File: C-CPP-Java-Logic-Building/11_Assignment/Assignment11_2.c
/* Programm to accept a number from user di andsplay pattern
    input = 5 
    output- 5 # 4 # 3 # 2 # 1 #
*/

#include<stdio.h>

int Pattern(int iNo)
{
    int iDigit = 0;
    int iCnt = 0;

    if (iNo < 0)
    {
        iNo = -iNo;
    }

    for(iCnt = iNo; iCnt >= 1; iCnt--)
    {
        printf("%d \t # \t",iCnt);   
    }

}

int main()
{
    int iValue = 0, iRet = 0;

    printf("Enter the number : ");
    scanf("%d", &iValue);

    Pattern(iValue);

    return 0;
}

File: Python-Logic-Building/11_Assignment/Assignment11_2.py
fact = 1

def Fact(no):
    global fact
    if(no >= 1):
        fact = fact * no
        no = no - 1
        Fact(no)
    return fact

def main():
    num = int(input("Enter number : "))
    iRet = Fact(num)
    print("Factorial of %d is :%d "% (num,iRet))

if __name__ == "__main__":
    main()